src/mlpack/methods/lars/lars.cpp

```diff
@@ -538,3 +538,20 @@ void LARS::CholeskyDelete(const size_t colToKill)
     matUtriCholFactor.shed_row(n);
   }
 }
+
+double LARS::ComputeError(const arma::mat& matX,
+                          const arma::rowvec& y,
+                          const bool rowMajor)
+{
+  double cost = 0.0;
+  arma::rowvec u;
+
+  if (rowMajor)
+    u = trans(matX * betaPath.back());
+
+  else
+    u = betaPath.back().t() * matX;
+
+  cost = arma::accu((y - u) % (y - u));
```

Comment:
Looks like we can just return the result instead of storing it into the `cost` parameter, also not sure, but I guess: `arma::accu(arma::pow(y - u, 2.0))` could be slightly faster. Also I guess since `u` would be used only once, we could move this into the if/else case.

Reply:
Done @zoq